plonky2x/src/frontend/vars/byte.rs

```diff
@@ -118,6 +121,22 @@ impl ByteVariable {
 
         [ByteVariable(left_nibble), ByteVariable(right_nibble)]
     }
+
+    pub fn to_variable<L: PlonkParameters<D>, const D: usize>(
+        self,
+        builder: &mut CircuitBuilder<L, D>,
+    ) -> Variable {
+        let _two = builder.constant::<Variable>(L::Field::from_canonical_u8(2));
+        let mut acc = builder.zero::<Variable>();
```

Comment:
@jtguibas is this the preferred way for getting `zero` and `one` as constants? I'm not sure